librz/main/rz-ax.c

```diff
@@ -613,7 +647,7 @@ static int rax(RzNum *num, char *str, int len, int last, ut64 *_flags, int *fm)
 
 		return true;
 	} else if (has_flag(flags, RZ_AX_FLAG_BIN_TO_BIGNUM)) { // -L
-		print_hex_from_base2(str);
+		print_hex_from_base2(rz_ax_parse_bitstring(str));
```

Comment:
i do not think you have tested this. i strongly doubt that you need to pass the converted string to `print_hex_from_base2`
just print the returned string unless is NULL.

Reply:
> i do not think you have tested this. i strongly doubt that you need to pass the converted string to `print_hex_from_base2` just print the returned string unless is NULL.

I do it. but now ```print_hex_from_base2``` defined but not used .

Reply:
delete it